src/ProductOptionValueAccessControlHandler.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Drupal\commerce_option;
```

Comment:
Same with this file, in the future we can refactor the file to be under, `/src/Entity/AccessControlHandler`

Reply:
Thanks.